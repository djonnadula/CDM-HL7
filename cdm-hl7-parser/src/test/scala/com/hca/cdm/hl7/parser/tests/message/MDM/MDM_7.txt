MSH|^~\&||AKAR|||20160930215531|PAQ8864|MDM^T02|EPIC_AKAR_MDM_12928085|P|2.1||Epic_Mountain_PRD|||||||
EVN|T02|20160930215531||UPDATE_NOTE|PAQ8864^LIU^LUKE^D^^^^^MTN A^^^^^AKAR
PID|||1001664262^^^EPIC^MRN||NERO^BERNADETTE||19670810|F||B|12920 FOSTER RD^^ANCHORAGE^AK^99516^USA^P^^ANCHORAGE|ANCHORAGE|(907)644-8242^P^H||ENGLISH|M|PRO|700323426|777-77-7777|||HISPANIC||||||||N
PD1||||VFK2116^KROPP^LAWRENCE^A^^^^^THREEFOUR^^^^THREEFOUR||||||||||||
PV1||I|AKARORTH^L.ORW732^L.ORW732A^AKAR^1^^^^^^|||||||ORTH|||||||RCU9895^VERMILLION^DOUG^A^^^^^THREEFOUR^^^^THREEFOUR||22635411|||||||||||||||||||||||^^^AKAR^^^^^^^||20160930||||||||


