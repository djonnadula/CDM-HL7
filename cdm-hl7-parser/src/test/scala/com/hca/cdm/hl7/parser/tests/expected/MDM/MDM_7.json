{
  "0000.COMN": {
    "sending_facility": "AKAR",
    "message_control_id": "EPIC_AKAR_MDM_12928085",
    "medical_record_num": "1001664262",
    "medical_record_urn": "",
    "patient_account_num": "700323426"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_facility": "AKAR",
    "date_time_of_message": "20160930215531",
    "security": "PAQ8864",
    "message_type": {
      "message_code": "MDM",
      "trigger_event": "T02"
    },
    "message_control_id": "EPIC_AKAR_MDM_12928085",
    "processing_id": "P",
    "version_id": "2.1",
    "continuation_pointer": "Epic_Mountain_PRD"
  },
  "0002.EVN": {
    "event_type_code": "T02",
    "recorded_date_time": "20160930215531",
    "event_reason_code": "UPDATE_NOTE",
    "operator_id": {
      "person_id": "PAQ8864",
      "family_name": "LIU",
      "given_name": "LUKE",
      "second_and_further_given_names_or_initials_thereof": "D",
      "assigning_authority": "MTN A",
      "assigning_facility": "AKAR"
    }
  },
  "0003.PID": {
    "patient_id_list": {
      "id_num": "1001664262",
      "assigning_authority": "EPIC",
      "id_type_code": "MRN"
    },
    "patient_name": {
      "family_name": "ABZU",
      "given_name": "CAROLINE"
    },
    "date_time_of_birth": "19880808",
    "administrative_sex": "F",
    "race": "B",
    "patient_address": {
      "street_address": "12 RD",
      "city": "TOWN",
      "state_or_province": "AK",
      "zip_or_postal_code": "88888",
      "country": "USA",
      "address_type": "P",
      "county_parish_code": "TOWN"
    },
    "county_code": "TOWN",
    "phone_num_home": {
      "telephone_num": "(999)999-9999",
      "telecommunication_use_code": "P",
      "telecommunication_equipment_type": "H"
    },
    "primary_language": "ENGLISH",
    "marital_status": "M",
    "religion": "PRO",
    "patient_account_num": "700323426",
    "ssn_num_patient": "777-77-7777",
    "ethnic_group": "HISPANIC",
    "patient_death_indicator": "N"
  },
  "0004.PD1": {
    "patient_primary_care_provider_name_and_id_num": {
      "person_id": "VFK2116",
      "family_name": "KROPP",
      "given_name": "LAWRENCE",
      "second_and_further_given_names_or_initials_thereof": "A",
      "assigning_authority": "THREEFOUR",
      "id_type_code": "THREEFOUR"
    }
  },
  "0005.PV1": {
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "AKARORTH",
      "room": "L.ORW732",
      "bed": "L.ORW732A",
      "facility": "AKAR",
      "location_status": "1"
    },
    "hospital_service": "ORTH",
    "admitting_doctor": {
      "person_id": "RCU9895",
      "family_name": "VERMILLION",
      "given_name": "DOUG",
      "second_and_further_given_names_or_initials_thereof": "A",
      "assigning_authority": "THREEFOUR",
      "id_type_code": "THREEFOUR"
    },
    "visit_num": "22635411",
    "pending_location": {
      "facility": "AKAR"
    },
    "admit_date_time": "20160930"
  }
}